package-lock.json

```diff
@@ -4628,8 +4628,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
You'll still want to revert this file. Probably the easiest way to do this is to make sure you are on the latest version of `npm`. I would suggest running these commands (assuming you have `nvm`, lmk if you don't):
`nvm install 10.15.3 && npm install -g npm@latest && npm install` which will make sure the `package-lock.json` will get switched back to the original format.

Otherwise you can pull out that file change from your first commit, but that will probably be more of a pain imo.

Reply:
Let me know if you want to pair on fixing this together, happy to help!